tests/api-tests/fields/types/fixtures/decimal/test-fixtures.ts

```diff
@@ -55,7 +55,7 @@ export const crudTests = (keystoneTestWrapper: any) => {
       expect(result.errors).toHaveLength(1)
       expect(result.errors![0].message).toMatchInlineSnapshot(`
         "You provided invalid data for this operation.
-          - Test.price: Price must be greater than or equal to -300"
+          - Test.price: value must be greater than or equal to -300"
```

Comment:
Errors are already prefixed with `{listKey}.{fieldKey}`, this humanized part was redundant and became difficult to understand for longer names